### PR TITLE
feature: Added match room map estimations

### PR DESCRIPTION
**Pull request by Morabotti**
Implementation of #10

### CODE REVIEW

File: src/content/helpers/faceit-api.js [hunk 1]
```diff
@@ -63,9 +63,9 @@ export const getUser = userId => fetchApiMemoized(`/core/v1/users/${userId}`)
 export const getPlayer = nickname =>
   fetchApiMemoized(`/core/v1/nicknames/${nickname}`)
 
-export const getPlayerMatches = (userId, game, size = 21, page = 0) =>
+export const getPlayerMatches = (userId, game, size = 20) =>
   fetchApiMemoized(
-    `/stats/v1/stats/time/users/${userId}/games/${game}?page=${page}&size=${size}`
+    `/stats/api/v1/stats/time/users/${userId}/games/${game}?size=${size}`
```
Comment (timche) [inline src/content/helpers/faceit-api.js:68]: Where did you get this endpoint from? Afaik this is not how FACEIT are naming their endpoints.

File: src/content/helpers/maps.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+export default {
+  de_dust2: 'Dust2',
+  de_mirage: 'Mirage',
+  de_overpass: 'Overpass',
+  de_inferno: 'Inferno',
+  de_nuke: 'Nuke',
+  de_cache: 'Cache',
+  de_train: 'Train',
+  de_vertigo: 'Vertigo'
+}
```
Comment (timche) [inline src/content/helpers/maps.js:11]: Import naming could be made more consistent:
```suggestion
export default {
	csgo: {
	  de_dust2: 'Dust2',
	  de_mirage: 'Mirage',
	  de_overpass: 'Overpass',
	  de_inferno: 'Inferno',
	  de_nuke: 'Nuke',
	  de_cache: 'Cache',
	  de_train: 'Train',
	  de_vertigo: 'Vertigo'
  }
}
```

```ts
import maps from '../helpers/maps';

// maps.csgo
```

File: src/content/helpers/faceit-api.js (hunk 1 above)
Comment (Morabotti) [reply src/content/helpers/faceit-api.js:68]: I changed this to fix memoization. 

Line 86 uses this API too:
>  let averageStats = await fetchApiMemoized(
    `/stats/api/v1/stats/time/users/${userId}/games/${game}?size=${size}`
  )

I shall change them both to make sure that there are no multiple requests done.

**Comment by ajohnsen** [comment]
Any chance of getting this one approved? :)

**Comment by Morabotti** [comment]
Changed based on feedback: https://github.com/faceit-enhancer/faceit-enhancer/issues/10#issuecomment-846378246. Here is a image of a new ui: 
veto

**Review by poacher2k — APPROVED** [review]
LGTM 👍

**Pull request merged**